INSTALL.md

```diff
@@ -54,17 +61,49 @@ When compiling on Ubuntu 16.04 or on Debian stretch, the following error can occ
                  poll_list[old_poll_nb].socket = *tmp_sock;
 ```
 
-This is due to incompatibility of zmq.hpp file provided in libzmq3-dev:4.1.7 (ubuntu 16.04), i.e. it is not possible to compile cppTango using zmq.hpp file provided by libzmq3-dev:4.1.7 (ubuntu 16.04).
+This is due to incompatibility of zmq.hpp file provided in libzmq3-dev:4.1.7
+(ubuntu 16.04), i.e. it is not possible to compile cppTango using zmq.hpp file
+provided by libzmq3-dev:4.1.7 (ubuntu 16.04).
+
+This requires installing [cppzmq](https://github.com/zeromq/cppzmq) via:
+
+```
+git clone https://github.com/zeromq/cppzmq
+cd cppzmq
+mkdir build
+cd build
+cmake -DENABLE_DRAFTS=OFF -DCPPZMQ_BUILD_TESTS=OFF ..
+sudo make install
+```
+
```

Comment:
Not sure if always required, but we also did these steps on my Ubuntu 16.04 VM:
- edit `/etc/apt/sources.list` - uncomment `deb-src` lines for `main restricted` and `universe`
- `sudo apt-get update`
- `sudo apt-get build-dep libtango9`

Reply:
@ajoubertza Thx. Added.